### PR TITLE
Cypress test. Autoborder handler.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Add Cypress test.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_tasks_objects/case_60_autoborder_feature.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    before(() => {
+        cy.openTaskJob(taskName);
+        cy.createRectangle(createRectangleShape2Points);
+        cy.createRectangle(createRectangleShape2PointsSec);
+
+        // Collect the rectagle points coordinates
+        testActivatingShape(450, 400, '#cvat_canvas_shape_1');
+        testCollectCxCircleCoord(rectangleSvgJsCircleId);
+        testActivatingShape(650, 400, '#cvat_canvas_shape_2');
+        testCollectCxCircleCoord(rectangleSvgJsCircleIdSecond);
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_60_autoborder_feature.js:74]: Should we test anything in before?

File: tests/cypress/integration/actions_tasks_objects/case_60_autoborder_feature.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            cy.get('.cvat-canvas-container').click(400, 350).click(450, 250).click(500, 350).click(500, 450);
+            cy.get('.cvat-canvas-container').trigger('keydown', { keyCode: keyCodeN }).trigger('keyup');
+            cy.get('.cvat_canvas_autoborder_point').should('not.exist');
+
+            // Collect the polygon points coordinates
+            testActivatingShape(450, 300, '#cvat_canvas_shape_3');
+            testCollectCxCircleCoord(polygonSvgJsCircleId);
+        });
+
+        it('Start drawning a polyline with autobordering beetwen the two shapes.', () => {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_60_autoborder_feature.js:92]: drawing*

File: tests/cypress/integration/actions_tasks_objects/case_60_autoborder_feature.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_60_autoborder_feature.js]: Moved to the 1st it(). These functions simply collect parameter values.

File: tests/cypress/integration/actions_tasks_objects/case_60_autoborder_feature.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_60_autoborder_feature.js]: Typo fixed.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**